core/domain/suggestion_services_test.py

```diff
@@ -1086,6 +1086,10 @@ def setUp(self):
         self.author_id_2 = self.get_user_id_from_email(self.AUTHOR_EMAIL_2)
         self.signup(self.REVIEWER_EMAIL_2, 'reviewer2')
         self.reviewer_id_2 = self.get_user_id_from_email(self.REVIEWER_EMAIL_2)
+        self.opportunity_summary_ids = [self.explorations[0].id,
```

Comment:
Prefer breaking after [

Reply:
Done.

Reply:
I would've thought what was meant was something like:

```
self.opportunity_summary_ids = [
  self.explorations[0].id, self.explorations[1].id, self.explorations[2].id]
```

Reply:
Yup, what @sagangwee suggested is what I meant.

Please see the guidelines [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#python) with regards to indentation.